lib/tools.js

```diff
@@ -177,18 +177,18 @@ function asChindings (instance, bindings) {
 function getPrettyStream (opts, prettifier, dest, instance) {
   if (prettifier && typeof prettifier === 'function') {
     prettifier = prettifier.bind(instance)
-    return prettifierMetaWrapper(prettifier(opts), dest)
+    return prettifierMetaWrapper(prettifier(opts), dest, opts.suppressFlushSyncWarning)
   }
   try {
     var prettyFactory = require('pino-pretty')
     prettyFactory.asMetaWrapper = prettifierMetaWrapper
```

Comment:
I also wanted to ask why we don't `bind` instance here?

Reply:
I don’t understand, how would you bind() it?

Reply:
Same way as at 179 `prettyFactory = prettyFactory.bind(instance)`.
`prettyFactory` seems to be the same function as `prettifier`

Reply:
I still don't understand why would you bind it?

Reply:
Ah, why. Just to conform to prettier API, 

> The uninitialized Pino instance is passed as this into prettifier factory function, so it can be accessed via closure by the returned prettifier function.